disasterinfosite/models.py

```diff
@@ -333,80 +333,89 @@ def getRelatedTemplate(self):
     def findSnuggetsForPoint(lat=0, lng=0, merge_deform = True):
         pnt = Point(lng, lat)
 
-######################################################
-# GENERATED CODE GOES HERE
-# DO NOT MANUALLY EDIT CODE IN THIS SECTION - IT WILL BE OVERWRITTEN
-# modelsGeoFilters
```

Comment:
Yeah, I decided to take that block out because I'm deliberately changing what gets generated. I think it would be better to make the changes here by hand from this juncture, if it's feasible.